.storybook/main.js

```diff
@@ -4,7 +4,18 @@ const path = require("path");
 const fs = require("fs");
 
 module.exports = {
-    stories: ["../packages/**/*.@(stories|fixturestories).@(js|jsx|ts|tsx)"],
+    stories: [
+        // NOTE(jeremy): This glob is extremely finicky! I would have written
+        // this as a negated match to exclude node_modules, but I was never
+        // able to get it to work. For example, the following regex included
+        // stories from wonder-blocks packages in node_modules.
+        //     "../packages!(/node_modules)/**/*@(.stories|.fixturestories).@(js|jsx|ts|tsx|mdx)",
+        // So, instead of fighting it, I changed this glob to restrict stories
+        // to be ones in any of our local packages 'src' dirs. This effectively
+        // eliminates stories showing up inside node_modules within any package
+        // dir.
+        "../packages/*/src/**/*@(.stories|.fixturestories).@(js|jsx)",
```

Comment:
Most of the time `yarn` installs deps in the root `node_modules` when using yarn workspaces, except when it doesn't. 😞

Reply:
Well, to be fair, these WB reps are in `packages/perseus/package.json`. But yeah, I wonder if `pnpm` handles this better?

Reply:
I'm curious about `pnpm` too.  I've used in a couple of repos where I was experimenting with other things, but I've never tried it with a mono-repo setup.